load_game.py

```diff
@@ -8,6 +8,7 @@
 import logs
 import memory.main
 import pathing
+import reset
```

Comment:
What's going on here? Missing import? Or accidental added import?

Reply:
Line 41 (not modified by me):
```
    if memory.main.get_map() not in [23, 348, 349]:
        reset.reset_to_main_menu()
```